pkg/groups/service.go

```diff
@@ -357,30 +362,24 @@ func (s *Service) DeleteGroup(ctx context.Context, ID string) error {
 	return nil
 }
 
-// ListIdentities returns all the identities (users for now) assigned to a group
-func (s *Service) ListIdentities(ctx context.Context, ID, continuationToken string) ([]string, string, error) {
+// ListIdentities returns all the identities (users and group#member associated) assigned to a group
+func (s *Service) ListIdentities(ctx context.Context, ID string) (*ListIdentitiesResult, error) {
 	ctx, span := s.tracer.Start(ctx, "groups.Service.ListIdentities")
 	defer span.End()
 
-	r, err := s.ofga.ReadTuples(ctx, "", authz.MEMBER_RELATION, authz.GroupForTuple(ID), continuationToken)
-
+	users, err := s.ofga.ListUsers(ctx, "user", authz.MEMBER_RELATION, authz.GroupForTuple(ID))
 	if err != nil {
 		s.logger.Error(err.Error())
-		return nil, "", err
+		return nil, err
 	}
 
-	identities := make([]string, 0)
-
-	for _, t := range r.GetTuples() {
-		// TODO @shipperizer the user: bit will have to change when or if we use the identity type, this will be tricky
-		// best way right now might be to verify if a user is also an identity (no idea how though)
-		// at the moment an identity cannot be a member of a group, only a user
-		if strings.HasPrefix(t.Key.User, "user:") {
-			identities = append(identities, t.Key.User)
-		}
+	groups, err := s.ofga.ListUsers(ctx, "group#member", authz.MEMBER_RELATION, authz.GroupForTuple(ID))
```

Comment:
ok i mulled it over a bit, i d be more inclined to stick with one `[]string` made of identities and groups as before for the time being

doesn't trigger a ui change and from a method signature us more inline with the others

Reply:
Even though I do not agree with this, after an offline sync we will address this and postpone changes to the API